.github/workflows/ml.yaml

```diff
@@ -30,7 +30,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest]
+        os: [ubuntu-latest, ubuntu-nightly]
```

Comment:
I see how you got to this solution, because the conda-envs are called from the os matrix. However that was just to avoid me typing it all out. 

This won't work because os is a reserved keyword for the runner type. You need something like the below

```yaml
matrix:
  os: [ubuntu-latest]
  conda-env: [ubuntu-latest, ubuntu-latest-pyg-nightly]


    - name: Setup Conda Environment
      uses: mamba-org/setup-micromamba@v1
      with:
        environment-file: devtools/conda-envs/asapdiscovery-${{ matrix.conda-env }}.yml
        environment-name: asapdiscovery
        cache-environment: true
        cache-downloads: true
        cache-environment-key: environment-${{ steps.date.outputs.date }}
        cache-downloads-key: downloads-${{ steps.date.outputs.date }}
        create-args: >-
          python==${{ matrix.python-version }}

```

I wouldn't bother with a macos nightly build too painful.

Reply:
Thanks!